DragonSlayer.h

```diff
@@ -1,8 +1,21 @@
 #pragma once
+#include "Character.h"
+#include "AttackItem.h"
 
 struct DragonSlayer : Character
 {
+    DragonSlayer(std::string , int, int ) ; 
     void attack(Character& other) override;
+    const std::string& getName() override;  // rewrite the inherited getName
+    std::string getStats() override;        // rewrite the inherited getStats
+    std::unique_ptr<AttackItem> attackItem = std::make_unique<AttackItem>();  // special Slayer attack item
```

Comment:
No one else needs to know about this object so it does not need to be public. Make it private
- [ ] revise